dashboard/legacy/test/middleware/test_files.rb

```diff
@@ -711,13 +712,14 @@ def test_files_copy_all
     assert_equal sound_body, last_response.body
 
     # abuse score didn't carry over
+    # note: this test doesn't actually work, since the abuse score functionality
+    # got moved to Rails.
     assert_equal 0, FileBucket.new.get_abuse_score(dest_channel_id, CGI.escape(image_filename.downcase))
     assert_equal 0, FileBucket.new.get_abuse_score(dest_channel_id, escaped_sound_filename.downcase)
 
     assert_newrelic_metrics %w(
       Custom/ListRequests/FileBucket/BucketHelper.app_size
       Custom/ListRequests/FileBucket/BucketHelper.app_size
-      Custom/ListRequests/FileBucket/BucketHelper.list
```

Comment:
this line and the change in the vcr recording are because of the missing `patch_abuse` call.